src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 17px;
+}
+
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+h2 {
+  font-weight: normal;
+  font-size: 15px;
+  margin: 1em 0;
+}
+
+a {
+  color: green;
+}
```

Comment:
Please use classes, not tags to add styles.
However, font-family and font-size can be kept in `body` selector.